nodejs/npmmirror.lua

```diff
@@ -0,0 +1,145 @@
+---  Default global variable
+---  OS_TYPE:  windows, linux, darwin
+---  ARCH_TYPE: 386, amd64, arm, arm64  ...
+local http = require("http")
+local json = require("json")
+
+OS_TYPE = ""
+ARCH_TYPE = ""
+
+NodeBaseUrl = "https://cdn.npmmirror.com/binaries/node/v%s/"
+FileName = "node-v%s-%s-%s%s"
+npmDownloadUrl = "https://cdn.npmmirror.com/binaries/node/npm/v%s.%s"
+
+VersionSourceUrl = "https://cdn.npmmirror.com/binaries/node/index.json"
+
+PLUGIN = {
+    name = "nodejs",
+    author = "Aooohan",
+    version = "0.0.2",
```

Comment:
For new plugins. The initial version number should be 0.0.1.
```suggestion
    version = "0.0.1",
```